plat/renesas/rcar/include/platform_def.h

```diff
@@ -194,7 +194,7 @@
 #elif IMAGE_BL2
 # define MAX_XLAT_TABLES		3
 #elif IMAGE_BL31
-# define MAX_XLAT_TABLES		4
+# define MAX_XLAT_TABLES		5
```

Comment:
four tables is not enough? You Just added one more mapping. Does it use whole table?

Reply:
Yes, is not enough. ARM TF started complaining about it right after I had added new mapping.